spec/models/ext_management_system_spec.rb

```diff
@@ -25,6 +25,7 @@
       scvmm
       vmwarews
       azure
+      ansible_tower_configuration
```

Comment:
Why is the configuration at the end?  Does it provide other services besides that?  I only ask because it's different that the other ones here (which might be a good thing anyway if it more closely matches the class names).